source/atlas-data-lake-testing/tests/README.rst

```diff
@@ -0,0 +1,66 @@
+=====================
+Atlas Data Lake Tests
+=====================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to assert compatibility with `Atlas Data Lake <https://docs.mongodb.com/datalake>`_.
+
+Running these integration tests will require a running ``mongohoused``
+with data available in its ``test.driverdata`` collection. See the
+`ADL directory in drivers-evergreen-tools <https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/atlas_data_lake>`_
+and `10gen/mongohouse README <https://github.com/10gen/mongohouse/blob/master/README.md>`_
+for more information.
+
+Several prose tests, which are not easily expressed in YAML, are also presented
+in this file. Those tests will need to be manually implemented by each driver.
+
+Test Format
+===========
+
+The same as the `CRUD Spec Test format <../../crud/tests/README.rst#Test-Format>`_.
+
+Prose Tests
+===========
+
+The following tests MUST be implemented to fully test compatibility with
+Atlas Data Lake.
+
+#. Test that the driver properly constructs and issues a
+   `killCursors <https://docs.mongodb.com/manual/reference/command/killCursors/>`_
+   command to Atlas Data Lake. For this test, configure an APM listener on a
+   client and execute a query that will leave a cursor open on the server (e.g.
+   specify ``batchSize=2`` for a query that would match 3+ documents). Drivers
+   MAY iterate the cursor if necessary to execute the initial ``find`` command
+   but MUST NOT iterate further to avoid executing a ``getMore``.
+
+   Observe the CommandSucceededEvent event for the ``find`` command and extract
+   the cursor's ID and namespace from the response document's ``cursor.id`` and
+   ``cursor.ns`` fields, respectively. Destroy the cursor object and observe
+   a CommandStartedEvent and CommandSucceededEvent for the ``killCursors``
+   command. Assert that the cursor ID and target namespace in the outgoing
+   command match the values from the ``find`` command's CommandSucceededEvent.
+   Assert that the ``killCursors`` CommandSucceededEvent indicates that the
+   expected cursor was killed in the ``cursorsKilled`` field.
```

Comment:
My [previous comment](https://github.com/mongodb/specifications/pull/821#discussion_r453078602) referred to executing a manual `getMore`, but I actually don't think that's relevant to drivers. Inspecting the CommandSucceededEvent for `killCursors` is sufficient to assert that ADL killed the cursor.

Also, if a driver did manually execute a `getMore` and got the namespace/ID wrong, the end result would be no different than if we got it right and the cursor was successfully killed.